src/index.d.ts

```diff
@@ -26,26 +26,26 @@ export interface FluxStandardAction<Payload, Meta> {
   meta: Meta;
```

Comment:
Isn't this supposed to be optional?

Reply:
Nope. That's the whole point. It should not be optional. See #53 and related discussion.

Reply:
Hey, sorry to butt in here. Just hit this issue and am a bit confused... Currently I have this issue:

```ts
const x: FSA<string, undefined> = { type: "foo", payload: "hello" }
```
```
Type '{ type: string; payload: string; }' is not assignable to type 'FluxStandardAction<string, undefined>'.
  Property 'meta' is missing in type '{ type: string; payload: string; }'.
```

So setting the type to `undefined` does not make the attribute optional.

I think the DefinitelyTyped defs handle this [in a more appropriate way](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7f0c54c13065b2a7f8e26a9fcc60e8b738916a77/types/flux-standard-action/index.d.ts#L18).

Reply:
We discussed that here: https://github.com/acdlite/flux-standard-action/pull/53#issuecomment-279258673

It is debatable on which is a better solution.
I would say when https://github.com/Microsoft/TypeScript/issues/12400#issuecomment-296360175 is resolved, we will be just fine.

Reply:
Ah yeah, makes sense. It's just annoying having to add extra code to satisfy types, but a small price to pay I suppose. :)